MODULES/EDGE.py

```diff
@@ -2567,16 +2580,16 @@ def add_spectra(self, scope, wlarr, fluxarr, errors=None, py2 = False, clob = 1)
                 self.spectra[scope] = {'wl': wlarr, 'lFl': fluxarr, 'err': errors}
         return
 
-    def add_photometry(self, scope, wlarr, fluxarr, errors=None, ulim=0, verbose = 1, py2 = False):
+    def add_photometry(self, scope, wlarr, fluxarr, errors=None, ulim=None, verbose = 1, py2 = False):
         """
         Adds an entry to the photometry attribute.
 
         INPUTS
         scope: The telescope or instrument that the photometry was taken with.
         wlarr: The wavelength array of the data. Can also just be one value if an individual point. Should be in microns. Note: this is not checked.
-        fluxarr: The flux array corresponding to the data. Should be in erg s-1 cm-2. Note: this is not checked.
+        fluxarr: The flux array corresponding to the data. Can also just be one value if an individual point. Should be in erg s-1 cm-2. Note: this is not checked.
         errors: (optional) The array of flux errors. Should be in erg s-1 cm-2. If None (default), will not add.
-        ulim: BOOLEAN -- whether or not this photometric data is or is not an upper limit.
+        ulim: The upper limit array of the data. Can also just be one value if an individual point.
```

Comment:
A suggestion to clarify the format.

```suggestion
        ulim: The upper limit array (booleans) of the data, indicating which data points are upper limits. Can also just be one value if an individual point.
```